src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,56 @@
+.watch {
+  @include contentFlexCenterCenter;
+
+  width: $watch-size;
+  height: $watch-size;
+
+  background-color: #33867e;
+  box-shadow: 1px 5px 7px #1c1c1c;
+  border-radius: 50%;
+
+  &__center {
+    z-index: 1;
+
+    width: 0.03 * $watch-size;
+    height: 0.03 * $watch-size;
+
+    box-shadow: $elements-shadow;
+    background-color: $center-bg-color;
+    border: 2px solid $minute-color;
+    border-radius: 50%;
+  }
+
+  &__minutes {
+    @include placeOfHands;
+
+    width: 0.01 * $watch-size;
+    height: 0.4 * $watch-size;
+
+    box-shadow: $elements-shadow;
+    background-color: $minute-color;
+    border-radius: 10px;
+
+    @include rotateSettings(3660s);
```

Comment:
The rotation settings for the minute hand seem to be incorrect. As per the task, the minute hand should rotate every 60 seconds, not every 3660 seconds.